Services/News/NewsItemService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Andculture.Sitefinity.Testing.Models.Configuration;
+using Andculture.Sitefinity.Testing.Models.Content.News;
+
+namespace Andculture.Sitefinity.Testing.Services.News
+{
+    public class NewsItemService : ODataService<NewsItem>
+    {
+        public NewsItemService(TestingSettings testingSettings, string accessToken)
+            : base(testingSettings, accessToken)
+        {
+            EndpointUrl = "/sf/system/newsitems";
```

Comment:
Suggestion - Consider moving the path to a configuration store. That way we can quickly update it if it changes over time.